packages/integration-tests/src/test-project-setup/setup_dir_as_esm_module.ts

```diff
@@ -29,4 +29,20 @@ export const setupDirAsEsmModule = async (absoluteDirPath: string) => {
     stdio: 'inherit',
     cwd: absoluteDirPath,
   });
+
+  const pathsObj = {
+    // The path here is coupled with backend-function's generated typedef file path
+    '@env/*': ['../.amplify/function-env/*'],
```

Comment:
This is now the 3rd place where we define this path (other packages being `cli-core` and `backend-function`). I don't expect this number to grow but asking for opinions on if we should consolidate this in one place to remove the coupling